rest-api/dao/physical_measurements_dao.py

```diff
@@ -284,12 +292,21 @@ def _update_participant_summary(self, session, obj):
     participant_summary.physicalMeasurementsFinalizedTime = obj.finalized
     participant_summary.physicalMeasurementsCreatedSiteId = obj.createdSiteId
     participant_summary.physicalMeasurementsFinalizedSiteId = obj.finalizedSiteId
+    if 'physicalMeasurementsStatus' in obj:
+      participant_summary.physicalMeasurementsStatus = obj.physicalMeasurementsStatus
     participant_summary.lastModified = clock.CLOCK.now()
-    if participant_summary.physicalMeasurementsStatus != PhysicalMeasurementsStatus.COMPLETED:
+
+    if participant_summary.physicalMeasurementsStatus != PhysicalMeasurementsStatus.COMPLETED and \
+            participant_summary.physicalMeasurementsStatus != PhysicalMeasurementsStatus.CANCELLED:
+
       participant_summary.physicalMeasurementsStatus = PhysicalMeasurementsStatus.COMPLETED
       participant_summary_dao.update_enrollment_status(participant_summary)
       session.merge(participant_summary)
 
+    if participant_summary.physicalMeasurementsStatus == PhysicalMeasurementsStatus.CANCELLED:
```

Comment:
I think you can combine this with the if above; just make sure this runs whenever the participant summary physical measurement status changes.